### DISCUSSION TITLE
Provide an integration with Matomo Analytics so as for Posthog

### DISCUSSION THREAD

**Discussion by Chardonneaur** [langfuse/langfuse#1813] [Ideas]
### Describe the feature or potential improvement

Currently an integration between Langfuse and PostHog makes it possible to query a web analytics solution thanks to AI.
It would be great that Matomo which is a Free Software Web Analytics package too, could also be supported.
https://demo.matomo.cloud

**Comment by marcklingen** [comment]
thanks @Chardonneaur, heard great things about Matomo. Let's track interest (upvotes) and additional requirements/ideas regarding a potential integration here

**Comment by atom-box** [comment]
Matomo is opensource, which is great, but also consider it just for the power - Matomo is in some ways more feature-full than Google Analytics, or almost any web analytics solution, matching GA feature for feature since 2012, and adding features that GA is technically incapable of.

So Matomo would be a great partner for Langfuse AI

**Comment by Chardonneaur** [comment]
@marcklingen how can the Matomo team/community help/support you in developing this integration? 
Currently we have this community member, https://github.com/openmost, Ronan Hello, who developed two plugins one for ChatGPT and another for Mistral AI.
We know well our API, we are available for a call to show you or speed up anything we can.
Just let us know how we can help and we will.

**Reply by marcklingen** [reply]
thanks for reaching out, currently the posthog integration runs on non-public infra until [Langfuse v3](https://github.com/orgs/langfuse/discussions/1902) is merged. Building additional integrations/extensions will be easy once we are on v3 and the Posthog integration is in this repo.

**Reply by Chardonneaur** [reply]
@marcklingen if i can help in anything about Matomo, just let me know and i will be here. Any deadline for v3?

**Reply by marcklingen** [reply]
May/June, let's align once this is released

**Comment by Chardonneaur** [comment]
Hi @marcklingen a recent plugin has been developped in order to get a Swagger config file within Matomo https://github.com/openmost/Swagger. Would providing you with a Swagger config fil would help in the development of the integration of Langfuse?

**Reply by marcklingen** [reply]
currently this does not help yet, we need to wait for Langfuse v3 to add additional sync targets

**Reply by Chardonneaur** [reply]
any update? @marcklingen

**Reply by marcklingen** [reply]
There is no update on this yet unfortunately. It is still a planned addition to Langfuse to generalize the implementation and enable more exporters.
